From e0fc38352d9127efa47f88651a66ac1b6c10046f Mon Sep 17 00:00:00 2001
From: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Date: Fri, 30 Jun 2017 16:22:35 -0400
Subject: [PATCH 2581/2831] drm/amd/display: Remove unblanaced drm_vblank_put.

Since drm_vblank_get moved from this function to just
before do_flip no need to release vblank here in case of error.

Change-Id: Iedc900618da2a0c940f49fe63299c6ae3eec260d
Signed-off-by: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Reviewed-by: Tony Cheng <Tony.Cheng@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
Signed-off-by: Raveendra Talabattula <raveendra.talabattula@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
index 7e5dd52..509288c 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
@@ -1010,9 +1010,6 @@ static int amdgpu_atomic_helper_page_flip(struct drm_crtc *crtc,
 	return 0;
 fail:
 
-	if (ret)
-		drm_crtc_vblank_put(crtc);
-
 	drm_atomic_state_free(state);
 
 	return ret;
-- 
2.7.4

